FirebaseAppCheck.podspec

```
@@ -45,6 +44,7 @@ Pod::Spec.new do |s|
   s.osx.weak_framework = 'DeviceCheck'
   s.tvos.weak_framework = 'DeviceCheck'
 
+  s.dependency 'FirebaseAppCheckInterop', '~> 10.14'
```

should the minor instead be pinned to whatever the releasing version is? (likely 10.16)
```suggestion
  s.dependency 'FirebaseAppCheckInterop', '~> 10.16'
```

---

Yes, thanks for catching!